Modules/Sources/WordPressUI/FancyAlert/FancyAlertView.swift

```diff
@@ -62,7 +62,8 @@ open class FancyAlertView: UIView {
 
     /// TitleLabel: textColor
     ///
-    @objc public dynamic var titleTextColor: UIColor? {
+    /// Notice the "wp" prefix to avoid clashing with Apple's private API.
+    @objc public dynamic var wpTitleTextColor: UIColor? {
```

Comment:
Notice that this is unused in the apps. Happy to remove it in a follow up PR to avoid future confusion.